pyproject.toml

```diff
@@ -1,3 +1,69 @@
+[project]
+name ="ytdl-sub"
+dynamic = [ "version" ]
+authors = [ { name = "Jesse Bannon" } ]
+description = "Automate downloading metadata generation with YoutubeDL"
+readme = "README.md"
+requires-python = ">=3.10"
+license = { file = "LICENSE" }
+classifiers = [
+    "Topic :: Multimedia :: Sound/Audio",
+    "Topic :: Multimedia :: Video",
+    "License :: Public Domain",
+    "Environment :: Console",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+dependencies = [
+    "yt-dlp==2024.*",
```

Comment:
We probably want to keep this one as exact since it's susceptible to break ytdl-sub